Source/WebKit/UIProcess/RemoteLayerTree/RemoteAnimationStack.mm

```diff
@@ -214,6 +215,22 @@
 #endif
 }
 
+NSDictionary *RemoteAnimationStack::toJSONForTesting() const
```

Comment:
This remote animation code doesn't have to be Apple-platform specific (it's not in an Apple platform directory), and it's not other than this ObjC testing code. Consider not using ObjC types for the JSON stuff.